### ISSUE TITLE
Combobox is inaccessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6876]
## Environment

> Operating system

Windows 10 latest

> Browser

Edge, Firefox

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description

The Combobox has accessibility issues in Windows High Contrast Mode
- no chevron indicating a dropdown interaction
- the option matching the input text is not highlighted
- there is no focus indication in the listbox
![image](https://user-images.githubusercontent.com/40970507/93831750-2efeb000-fc39-11ea-8849-b66dd995fb95.png)

**Comment by tw15egan** [comment]
Seems to be resolved on latest 

![Screenshot 2021-03-04 181436](https://user-images.githubusercontent.com/11928039/110057472-7e332b80-7d15-11eb-84bc-292cbd0b881a.png)
